sdks/java/io/parquet/build.gradle

```
@@ -46,11 +47,10 @@ dependencies {
   compile "org.apache.parquet:parquet-common:$parquet_version"
   compile "org.apache.parquet:parquet-hadoop:$parquet_version"
   compile library.java.avro
+  compile "org.apache.parquet:parquet-column:1.10.0"
```

Also, please move this line so its with the rest of the parquet dependencies.

```suggestion
  compile "org.apache.parquet:parquet-column:$parquet_version"
```

---

Oops. I didn't notice the $parquet_version variable.